cmd/test_util.go

```diff
@@ -54,21 +54,22 @@ func runTest(t *testing.T, file string, function interface{}, errCode int, argSt
 			results = append(results, currentResult)
 		}
 	}
-	var errors []int
+	errors := map[int]bool{}
 	for _, result := range results {
 		for _, occurrence := range result.Occurrences {
-			errors = append(errors, occurrence.id)
+			errors[occurrence.id] = true
 		}
 	}
 
-	if errCode != 0 {
-		assert.Contains(errors, errCode)
+	for _, errCode := range errCodes {
```

Comment:
Nitpick: I would swap the two asserts because if the length is off you know it's not gonna work :)